ext/webidl/benches/dict.rs

```diff
@@ -12,15 +12,17 @@ use deno_core::ExtensionFileSourceCode;
 fn setup() -> Vec<Extension> {
   vec![
     deno_webidl::deno_webidl::init_ops_and_esm(),
-    Extension::builder("deno_webidl_bench")
-      .esm(vec![ExtensionFileSource {
+    Extension {
+      name: "deno_webidl_bench",
+      esm_files: std::borrow::Cow::Borrowed(&[ExtensionFileSource {
```

Comment:
If we can solve https://github.com/denoland/deno_core/issues/90, that would be a great cleanup of these legacy extensions.